pkg/storage/gcs/gcs.go

```diff
@@ -33,5 +38,25 @@ func (d *driver) Volumes() ([]corev1.Volume, []corev1.VolumeMount, error) {
 }
 
 func (d *driver) CompleteConfiguration() error {
+	if len(d.Config.Bucket) == 0 {
+		d.Config.Bucket = "image-registry"
+
+		projectID, err := metadata.NewClient(nil).ProjectID()
+		if err != nil {
+			return err
+		}
+
+		ctx := context.Background()
+
+		client, err := storage.NewClient(ctx)
+		if err != nil {
+			return err
+		}
+
+		err = client.Bucket(d.Config.Bucket).Create(ctx, projectID, nil)
```

Comment:
do you not need to check if the bucket already exists?

Reply:
That is why it's a simple gcs completion. I do not know how to do it better ))

Reply:
ok. So you're still investigating how to check if a bucket exists?

Reply:
@bparees Yes, I wanted to leave it as follow up. This PR is just a preview to understand how to update the configuration, when to do it, and how the function might looks like.

Reply:
that's reasonable